projects/swimlane/ngx-ui/src/lib/components/json-editor/schema-validator.service.ts

```diff
@@ -11,6 +11,24 @@ export class SchemaValidatorService {
     });
     this.ajv.addFormat('password', '.*');
     this.ajv.addFormat('code', '.*');
+
+    this.ajv.addKeyword('isNotEmpty', {
```

Comment:
We shouldn't break from the already defined specification like this. This could end up breaking our JSON Schema validation in other projects.
I'd prefer we find an alternative (something like [minLength](https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.6.3.2) or creating an additional format which is allowed in the spec)

Reply:
We'll close this PR... shemas will need to specify the minLength.

But does this really break JSON Schema validation?  Doesn't it support adding new validation keywords?